apps/docs/pages/docs/api-reference/data-model/app-state.mdx

```diff
@@ -98,12 +131,6 @@ Puck does not currently provide the UI to control this value, but it can be togg
 
 ---
 
-### `ui.rightSideBarVisible`
-
-Whether or not the right side bar is visible.
-
----
```

Comment:
I understand you reordered these to keep them group, but these docs are alphabetical.

If you would rather not worry about docs, feel free to drop these changes and we can handle in post.